tests/jobs/design2params/test_design2params.py

```diff
@@ -424,3 +424,67 @@ def test_pair_cell_values(cellvalues, expected_parameters_strs, tmpdir):
     key_vals_real1 = extract_key_value(params_lines_1)
     assert key_vals_real0["SOMEKEY"] == expected_parameters_strs[0]
     assert key_vals_real1["SOMEKEY"] == expected_parameters_strs[1]
+
+
+@pytest.mark.parametrize(
+    "paramnames",
+    (
+        [
+            ["REAL", "SOMEKEY "],
+            ["REAL", " SOMEKEY"],
+            ["REAL", " SOMEKEY "],
+            ["REAL", "\tSOMEKEY"],
```

Comment:
Tabs :heart:

Reply:
Not sure it is possible to even inject through Excel..